mappings.json

```diff
@@ -2404,6 +2404,7 @@
     "global": "Ember.RSVP.Promise",
     "module": "rsvp",
     "export": "Promise",
+    "localName": "EmberPromise",
```

Comment:
are we sure about this? I've always used `Promise` instead because I never wanted the native `Promise` and it's essentially a polyfill too 🤔 

and if we want to use a different name, why not `RSVPPromise` or `RSVPromise`? 😉

Reply:
Updated. I agree, I want to name it `Promise` for exactly the same reason.